Jropp/jason-ropp-js-iobp
# Onboarding Project: add-user component

## What It Does

This migrates the add-user form into a component and adds functionality, storing the users in local storage array under "localStorage.onboardProjectUsers"

The previous add-user form was not a polymer component and was not functional. It was simply a placeholder.

I used a function to get the form information and build then user. I then pulled up the database information from localStorage, pushed the new user to that array, then replaced the database with that updated array. 

add-user also checks to make sure all fields have been filled out before saving the file.

- Closes #30 
- Closes #31 

## How To Test

Click the 'Create User' Tab. Fill out the user form then click submit. `console.log(localStorage)` and you should see your newly created user in an array under the property `onboardUserDatabase`

The [user-list.html](https://github.com/Jropp/jason-ropp-js-iobp/blob/add-user/src/user-list/user-list.html) component dynamically adds database users to be displayed in users-list.html immediately. To see new users you have added, click on the 'User List' tab.

## Notes/Caveats

Currently only compatible with Chrome (will be addressed in #38)

user-list.html component will be mashed together/remade into a single component that allows editing, displaying, and creating a user in a later issue.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

## Dependencies

- [x] #33
